components/studio/bin/hab-studio.sh

```diff
@@ -844,14 +730,14 @@ info() {
   fi
 
   if [ "${HAB_NOCOLORING:-}" = "true" ]; then
-    printf -- "   ${program:-unknown}: $1\n"
+    echo "   ${program:-unknown}: $1"
```

Comment:
Using a raw `echo` breaks the ANSI coloring, requiring the `-e` flag which isn't always available--one reason why I used `printf` originally in `info()`, `exit_with()`, and `ensure_root()`